docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -667,13 +667,13 @@ After deploying these resources to Kubernetes, the Ingress resource will allow u
 
 To customize the default `host` and `path` properties of the generated Ingress resources, you need to apply the following configuration:
 
-[source]
+[source, properties]
 ----
 quarkus.kubernetes.ingress.expose=true
 # To change the Ingress host. By default, it's empty.
 quarkus.kubernetes.ingress.host=prod.svc.url
 # To change the Ingress path of the generated Ingress rule. By default, it's "/".
-quarkus.kubernetes.ports.http.path=/prod
+quarkus.kubernetes.ingress.path=/prod
```

Comment:
This property does not exist, the right one is `quarkus.kubernetes.ports.http.path`.

Reply:
Oh yes, I messed up with rule configuration.

By the way, it's not very easy to understand why a Kubernetes port have a path ?

Reply:
That's very true... We should address this not-very user-friendly property sooner than later.

Reply:
OK, I can open an issue.
Should I close the PR or just remove this change ? Other change are cosmetic.

Reply:
For me, it's fine to just remove this change and proceed with the rest. 
And yes, feel free to open an issue to improve this configuration.

Reply:
Line removed and issue create: https://github.com/quarkusio/quarkus/issues/29279